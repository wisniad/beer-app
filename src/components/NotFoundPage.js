import React from 'react';
import {Link} from 'react-router-dom';
import {Flex} from 'reflexbox'

const NotFoundPage = () => (
        <Flex
            align='center'
            justify='center'
            column
            w={1}
            px={1}
            py={2}>
            <h2>This is from 404 Not found page <Link to="/">Go to main page</Link></h2></Flex>
);

export default NotFoundPage;