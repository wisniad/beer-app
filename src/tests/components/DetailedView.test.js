import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import NotFoundPage from '../../components/NotFoundPage';


test('should render Header correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<NotFoundPage/>);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
    // console.log(renderer.getRenderOutput());
});