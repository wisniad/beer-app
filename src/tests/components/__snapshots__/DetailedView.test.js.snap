// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <Unknown />
  <GridWrapper
    container={true}
    spacing={24}
  >
    <GridWrapper
      align="center"
      item={true}
      xs={12}
    >
      <h2
        className="notfoundpage__paddingTop"
      >
        404 Not found page
      </h2>
      <h2>
        <Link
          className="notfoundpage_link"
          replace={false}
          to="/"
        >
          Go to main page
        </Link>
      </h2>
    </GridWrapper>
  </GridWrapper>
</div>
`;
